src/cli/README_HISTORY.md

```diff
@@ -292,6 +293,40 @@ Print only the latest 2 entries.
 Done
 ```
 
+### omrprefix
+
+Usage `history omrprefix [list] [<num-entries>]`
+
+Requires `OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE`.
+
+Print the favored OMR prefix history. Each entry provides:
+
+- The favored OMR prefix.
+- Preference (`high`, `med`, `low`).
+- IsLocal as boolean `yes`/`no` indicating whether the favored OMR prefix is the same as the local one maintained by this BR.
+
+Print the OMR Prefix history as a table.
+
+```bash
+> history omrprefix
+| Age                  | OMR Prefix                                       | Pref   |IsLocal |
++----------------------+--------------------------------------------------+--------+--------+
+|         00:00:10.110 | fd44:dc78:510b:1::/64                            | low    | yes    |
+|         00:06:17.604 | 2001:1a:12d5:23ae::/64                           | med    | no     |
+|         00:13:11.235 | fd44:dc78:510b:1::/64                            | low    | yes    |
+Done
+```
+
+Print the OMR Prefix history as a list
```

Comment:
```suggestion
Print the OMR prefix history as a list
```

Reply:
Thanks @Vyrastas. Updated all in new pushed commit.